include/boost/gil/image_processing/numeric.hpp

```diff
@@ -2,21 +2,40 @@
 
 namespace boost{
```

Comment:
coding style nit-pick: `namespace boost { namespace gil {` is more widely used across GILL code